cmd/convox/main.go

```diff
@@ -112,12 +112,11 @@ func currentRack(c *cli.Context) string {
 func rackClient(c *cli.Context) *client.Client {
 	host, password, err := currentLogin()
 	if err != nil {
-		stdcli.Error(err)
-		return nil
```

Comment:
At first, I went with this function returning an error. But as you mentioned, it's called everywhere in a chained fashion. All those calls would have to add error checking, which is fine in itself but then reduces the usefulness of this helper IMHO.

So I went with the function exiting out as the CLI can't do anything without a working client.